suite-native/module-settings/src/components/SettingsSectionItemText.tsx

```diff
@@ -0,0 +1,21 @@
+import { ReactNode } from 'react';
+
+import { Box, Text } from '@suite-native/atoms';
+
+export type SettingsSectionItemTextProps = {
+    title: ReactNode;
+    subtitle?: ReactNode;
+};
+
+export const SettingsSectionItemText = ({ title, subtitle }: SettingsSectionItemTextProps) => {
+    return (
+        <Box justifyContent="center" flexDirection="column" flex={2}>
```

Comment:
Nit: No need for specific flex values as `2`. The other siblings in the parent flexBox have no flex value set. `flex 1` should be sufficient.
```suggestion
        <Box justifyContent="center" flexDirection="column" flex={1}>
```

Reply:
https://github.com/trezor/trezor-suite/pull/14341/commits/86f647ce514f4d47df7c534495510902e9b80f66